usb_protocol/emitters/descriptors/standard.py

```diff
@@ -177,20 +177,23 @@ def get_index_for_string(self, string):
         return index
 
 
-    def add_descriptor(self, descriptor, index=0):
+    def add_descriptor(self, descriptor, descriptor_type=None, index=0):
```

Comment:
Can we add this after the `index` argument? Given index is allowed to be positional, it'd be nice not to break the API.